src/plugins/intel_gpu/thirdparty/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ if(ENABLE_ONEDNN_FOR_GPU)
         set(ONEDNN_BUILD_DIR "${CMAKE_CURRENT_BINARY_DIR}/onednn_gpu_build/")
         set(ONEDNN_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/onednn_gpu_install/")
         set(ONEDNN_PREFIX_DIR "${CMAKE_CURRENT_BINARY_DIR}/onednn_gpu_root")
+        set(ONEDNN_ENABLED_PRIMITIVES "BINARY;CONCAT;CONVOLUTION;DECONVOLUTION;ELTWISE;INNER_PRODUCT;MATMUL;REORDER;POOLING;PRELU;REDUCTION;SUM")
```

Comment:
Do we need to include `BINARY;ELTWISE;PRELU;SUM`? I think it is not necessary when those are used as post-operation.

Reply:
I've removed those ops from the list.